src/SFA.DAS.EAS.Employer_Account.Database/StoredProcedures/GetEmployerAgreementsToRemove_ByAccountId.sql

```diff
@@ -9,7 +9,9 @@ BEGIN
 			ALE.Name, 
 			ISNULL(Signed.StatusId, Pending.StatusId) AS Status, 
 			a.HashedId, 
-			le.Code
+			le.Code as LegalEntityCode,
+			le.[Source] as LegalEntitySource,
+			ale.PublicHashedId as HashedAgreementId
```

Comment:
That PublicHashedId  on ALE is not the agreement hashed id but the legal entity hashed id so this needs to be changed. I'm not even sure we have an agreement hashed ID in the database so you might need to unhash it and then compare against the id itself.

Reply:
in `GetEmployerAgreement.sql` it has the following:
`ale.PublicHashedId as AccountLegalEntityPublicHashedId`, which is the same as HashedAgreementId right?

Reply:
no, it's the hashed id of the legal entity that is associated with the agreement. As far as I can tell it is never used as the hashed agreement id